documentation/blog/2024-04-01-whats-new-in-svelte-april-2024.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "What's new in Svelte: April 2024"
+description: 'Svelte Summit Spring on April 27! Plus: Maps, Dates and Sets'
```

Comment:
maybe I'm overthinking this, but I wonder about rewording this slightly to leave off the s at the end of each word so that we use a word that matches the API
```suggestion
description: 'Svelte Summit Spring on April 27! Plus: reactive `Map`, `Date` and `Set`'
```